medicai/__init__.py

```
@@ -1,6 +1,19 @@
-from medicai.utils.general import hide_warnings
+import logging
+import os
+import warnings
+
+# Disable Tensorflow logging.
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+os.environ["TF_CPP_MIN_VLOG_LEVEL"] = "3"
+os.environ["TF_ENABLE_ONEDNN_OPTS"] = "0"
+
+# Disable Python warnings
+warnings.warn = lambda *args, **kwargs: None
```

Ruff (0.14.0)

Unused lambda argument: `args`

(ARG005)